pick.c

```diff
@@ -1041,13 +1063,55 @@ get_key(const char **key)
 	return PRINTABLE;
 }
 
+enum key
+get_vi_key(const char **key)
```

Comment:
Don't see the point of a pointer-pointer (no pun intended!). Probably due to copy-pasting the prototype for `get_key`.

Reply:
Of course that was the reason ... ;-)